addons/arsenal/functions/fnc_fillSort.sqf

```diff
@@ -0,0 +1,75 @@
+#include "script_component.hpp"
+#include "..\defines.hpp"
+
+params ["_display", "_control"];
+
+if (ctrlIDC _control == 17 && {GVAR(currentLeftPanel) in [IDC_buttonUniform ,IDC_buttonVest, IDC_buttonBackpack]}) then {
+    systemChat "running right";
+    private _sortRightCtrl = _display displayCtrl IDC_sortRightTab;
+    lbClear _sortRightCtrl;
+    private _handleStatsFnc = {
+        {
+            _sortLeftCtrl lbAdd (_x select 1);
+        } forEach GVAR(sortRightLeftPanel) select _this;
+    };
+    switch (GVAR(currentRightPanel)) do {
+        case IDC_buttonOptic: {
+            0 call _handleStatsFnc;
+        };
+        case IDC_buttonItemAcc: {
+            1 call _handleStatsFnc;
+        };
+        case IDC_buttonMuzzle: {
+            2 call _handleStatsFnc;
+        };
+        case IDC_buttonBipod: {
+            3 call _handleStatsFnc;
+        };
+        case IDC_buttonCurrentMag;
+        case IDC_buttonCurrentMag2;
+        case IDC_buttonMag;
+        case IDC_buttonMagALL: {
+            4 call _handleStatsFnc;
+        };
+        case IDC_buttonThrow: {
+            5 call _handleStatsFnc;
+        };
+        case IDC_buttonPut: {
+            6 call _handleStatsFnc;
+        };
+        case IDC_buttonMisc: {
+            7 call _handleStatsFnc;
+        };
+    };
+} else {
+    private _idc = ctrlIDC _control;
+    if ([IDC_buttonFace, IDC_buttonVoice, IDC_buttonInsigna] find _idc > -1) then {
```

Comment:
> IDC_buttonInsigna

looks like a typo?

Reply:
Looks like a typo from Alganthe, it's spelled that way everywhere